src/hooks/useAccountBalance.ts

```diff
@@ -88,25 +94,76 @@ export const useAccountBalance = ({
     },
   });
 
+  const { data: accounts } = useAccountGraz({
+    chainId: SUPPORTED_COSMOS_CHAINS,
+    multiChain: true,
+  });
+
+  const cosmosAddress = useMemo(() => {
```

Comment:
consider moving this to useAccounts

Reply:
Hmm I see that this requires a `chainId` to get the desired bech32.  I think this does complicate whether we move it to `useAccounts`. Maybe we only move the `grazAccounts` hook and rename this to `selectedChainAddress` instead of `cosmosAddress`